Documentation/ceph-upgrade.md

```diff
@@ -221,6 +221,31 @@ Replace the namespace names in the new resources:
 sed "s/ROOK_SYSTEM_NAMESPACE/$ROOK_SYSTEM_NAMESPACE/g" upgrade-from-v1.0-create.yaml > upgrade-from-v1.0-create.yaml.tmp
 sed "s/ROOK_NAMESPACE/$ROOK_NAMESPACE/g" upgrade-from-v1.0-create.yaml.tmp > upgrade-from-v1.0-create.yaml
 rm -f upgrade-from-v1.0-create.yaml.tmp
+sed -i "s/ROOK_NAMESPACE/$ROOK_NAMESPACE/g" upgrade-from-v1.0-apply.yaml
+```
+
+If you have a v1.0 cluster running with CSI drivers enabled, delete the rbac rules created for CSI
+
+```sh
+kubectl -n $ROOK_NAMESPACE delete serviceaccount/rook-csi-cephfs-plugin-sa
```

Comment:
Don't a number of these SAs and RBAC exist in v1.1. Why delete them? Can we shorten the list to only the ones that either don't exist in 1.1 or that have changed definitions? The `create` command below will just indicate the resource already exist for some of them, but that seems better than having a crazy long list here. The shorter we can make the instructions, the better.

Reply:
@travisn done